cloud-controller-manager/do/resources.go

```diff
@@ -53,7 +53,7 @@ type resources struct {
 }
 
 // newResources initializes a new resources instance.
-
+//
```

Comment:
Please keep this one in -- the upper and lower part of the comment belong to one "section".

Reply:
Do you mean that we should remove the line 56 completely or remove the change and keep the empty line?